packages/backend/src/ee/models/AiAgentModel.ts

```diff
@@ -1460,4 +1558,34 @@ export class AiAgentModel {
 
         return toolCallsAndResults;
     }
+
+    // TODO: review this
+    async hasUserAccessToAgentViaGroup(
```

Comment:
looks good. Though I guess this method assumes that group access is set on agent.

Reply:
oops, now that we have the other methods `setAndGetGroupAccess` and `getGroupAccess`  and `setGroupAccess` the method `hasUserAccessToAgentViaGroup` isn't necessary